inc/classes/class-preview-revisions.php

```diff
@@ -86,21 +91,23 @@ public function preview_revisions() {
 			}, 300 );
 		</script>
 		<?php
+
 	}
 
 	/**
 	 * Function to modify the post request.
 	 *
-	 * @param string $posts_request
-	 * @return string
+	 * @param string $posts_request Posts Request.
+	 *
+	 * @return string $posts_request Modified Posts Request.
 	 */
 	public function modify_posts_request( $posts_request ) {
```

Comment:
Hi @SH4LIN, Can you add the type checks here? we should use it.

`public function modify_posts_request( string $posts_request ): string`, Kind of this.

You can add it all places.

Reply:
I agree, But that would not be a sudden change by returning the array or string, Also this is our own-hooked function, and we are returning the string here. So I am thinking of to add this. 

Thanks.

Reply:
Sure, I'll update the code.

Thank you

Reply:
Thank you 🙂.